action.yml

```
@@ -18,15 +18,13 @@ inputs:
     description: Set to true in order to enable timestamps on log messages
     required: false
     default: 'true'
-  # TODO(user): Specify your inputs here
-  input-name:
-    description: Some input description
+  ecr-repo-name:
+    required: true
+  pr-number:
+    required: true
+  delete-untagged-resources:
     required: true
```

Our default behaviour or other CI platforms is the following:
```suggestion
  delete-untagged-resources:
    required: false
    default: "true"
```